packages/test/local-server-stress-tests/.eslintrc.cjs

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+	extends: [
+		require.resolve("@fluidframework/eslint-config-fluid/minimal-deprecated"),
+		"prettier",
+	],
+	rules: {
+		"@typescript-eslint/strict-boolean-expressions": "off", // requires strictNullChecks=true in tsconfig
```

Comment:
Where is the tsconfig for this package anyway?  Looks like for local-server-tests it pulls the test/tsconfig.json, is that appropriate or should there be a separate tsconfig for src/ like most of our other packages?